xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/misc.less

```diff
@@ -327,6 +327,7 @@ dl.export-tree-legend dt + dd {
 // Loading --------------------------------------------------------------------
 .loading {
   background: transparent url(@xwiki-icon-spinner) no-repeat center;
+  min-height: @xwiki-icon-spinner-height;
```

Comment:
I've introduced a new less variable to configure the spinner height. It is initialized to the height of the current spinner icon and needs to be synchronized if a new icon if `@xwiki-icon-spinner` is changed.
I guess the `.tab-content > .tab-pane.loading` block just below can be removed, but I'm not sure if it should be kept for legacy compatibility.

Reply:
> I guess the .tab-content > .tab-pane.loading block just below can be removed, but I'm not sure if it should be kept for legacy compatibility.

+1 to remove it.